cspell-config/cspell-zksync.txt

```diff
@@ -2,7 +2,7 @@
 boojum
 !MatterLabs
 Matter Labs
-!Hyperchain
+Hyperchain
```

Comment:
Why was `Hyperchain` added back?

Reply:
It's a temporary feature -- I'm fighting with cspell, but Hyperchain is used in some type names in SDK code.

Reply:
OK, it turns out that "Hyperchain" is used in our contract ([example](https://github.com/matter-labs/era-contracts/blob/aafee035db892689df3f7afe4b89fd6467a39313/l1-contracts/contracts/state-transition/chain-interfaces/IZkSyncHyperchain.sol)), and the usage in docs is valid (e.g. documentation for a method to return ABI of this contract), so I guess we have to allow it. I think by now it's unlikely that someone will accidentally use hyperchain out of context due to terminology being established.